app.js

```diff
@@ -5,29 +5,29 @@ const logger = require("morgan");
 const cors = require("cors");
 const swaggerUi = require("swagger-ui-express");
 const swaggerDocument = require("./swagger.json");
-const path = require('path');
+// const path = require('path');
```

Comment:
Please remove comments, the code should be clear. Or provide the reason why it should be commented.